File: app/src/main/java/com/example/fang/b16traveldomain/model/Coupon.java
package com.example.fang.b16traveldomain.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Coupon {

    @SerializedName("msg")
    @Expose
    private String msg;
    @SerializedName("discount")
    @Expose
    private String discount;

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public String getDiscount() {
        return discount;
    }

    public void setDiscount(String discount) {
        this.discount = discount;
    }

}

File: app/src/main/java/com/example/fang/b16traveldomain/model/Coupon.kt
package com.example.fang.b16traveldomain.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Coupon(@SerializedName("msg")
                       @Expose
                       var msg : String,
                       @SerializedName("discount")
                       @Expose
                       var discount :String)